src/mrsrequest/models.py

```diff
@@ -920,6 +920,72 @@ def save(self, *args, **kwargs):
             tries -= 1
 
 
+def remove_attachments_without_mrsrequest():
+    print('--- Removing MRSAttachments without MRSRequest ---')
+    try:
+        pmts = PMT.objects.filter(
+            pk__in=PMT.objects
+            .filter(
+                mrsrequest__isnull=True,
+                creation_datetime__lt=(
+                    datetime.datetime.now() - datetime.timedelta(
+                        days=8
+                    )
+                )
+            )
+            .values_list('pk', flat=True)
+        )
+        pmts_count = pmts.count()
+        pmts.delete()
+        print('Deleted {} PMTs'.format(pmts_count))
+    except Exception as e:
+        print('Error : {}'.format(e))
+
+    try:
+        bills = Bill.objects.filter(
+            pk__in=Bill.objects
+            .filter(
+                mrsrequest__isnull=True,
+                creation_datetime__lt=(
+                    datetime.datetime.now() - datetime.timedelta(
+                        days=8
+                    )
+                )
+            )
+            .values_list('pk', flat=True)
+        )
+        bills_count = bills.count()
+        bills.delete()
+        print('Deleted {} Bills'.format(bills_count))
+    except Exception as e:
+        print('Error : {}'.format(e))
```

Comment:
Tres bien, a noter qu'on pouvait aussi faire un `for Model in (Bill, PMT):` au lieu d'un c/c